### DISCUSSION TITLE
Welcome to lifeMEDeasy Discussions!

### DISCUSSION THREAD

**Discussion by avinashkranjan** [avinashkranjan/lifeMEDeasy#69] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arpit456jain** [comment]
hey @avinashkranjan I am Arpit I am a gssoc'21 participant I would like to contribute in this project  but i am facing some issues in running this project.
I had fork this repo then clone it into my system 
as there was no requirements.txt so I had not made a virtual env and just run py.exe manage.py runserver
and the server runs but i got an error of **template doesn't exist**

Am I missing something , kindly help me out
